src/envelope.rs

```diff
@@ -48,11 +50,46 @@ pub trait MessageEnvelope: Send {
     ) -> BoxFuture<'a, ()>;
 }
 
+#[cfg(feature = "with-tracing-0_1")]
+#[derive(Clone)]
+struct Instrumentation {
+    parent: Span,
+    _in_queue: Span,
+}
+
+#[cfg(feature = "with-tracing-0_1")]
+impl Instrumentation {
+    fn new<A: Actor, M>() -> Self {
+        let parent = debug_span!(
+            parent: Span::current(),
+            "xtra actor request",
+            actor = std::any::type_name::<A>(),
+            message = std::any::type_name::<M>(),
+        );
+
+        // TODO rename: this is technically from send() not in queue alone. Counts waiting to get in
+        // to queue too.
+        let in_queue = debug_span!(
```

Comment:
Perhaps this is actionable now by renaming it to `waiting_for_actor`? (with an optional comment stating that it's the time from send() and being it queue together)

Reply:
Oops - I changed the name but forgot to change the variable and remove TODO :) Thanks!